mobile/android/android-application-manager.ts

```diff
@@ -25,27 +25,27 @@ export class AndroidApplicationManager implements Mobile.IDeviceApplicationManag
 			return this._installedApplications;
 		}).future<string[]>()();
 	}
-	
+
 	public installApplication(packageFilePath: string): IFuture<void> {
-		return this.adb.executeCommand(`install -r "${packageFilePath}"`);
+		return this.adb.executeCommand(["install", "-r", `${packageFilePath}`]);
```

Comment:
if packageFilePath contains spaces, we may still fail, maybe we should verify this and use something like:
`"${packageFilePath}"`